kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/model/instance/SolverInstance.java

```diff
@@ -87,18 +84,10 @@ public ScoreWrapper getScoreWrapper() {
         return scoreWrapper;
     }
 
-    public void setScoreWrapper( ScoreWrapper scoreWrapper ) {
+    public void setScoreWrapper(ScoreWrapper scoreWrapper) {
         this.scoreWrapper = scoreWrapper;
     }
 
-    public Object getPlanningProblem() {
-        return planningProblem;
-    }
-
-    public void setPlanningProblem(Object planningProblem) {
-        this.planningProblem = planningProblem;
-    }
-
```

Comment:
why delete planning-problem?
I use [POST] to execute optaplanner, but there occurs 500 Internal Server Error
they say there is no instance SolverInstance.planning-problem.